release notes/v0.50.0.md

```diff
@@ -0,0 +1,157 @@
+k6 `v0.50.0` is here 🎉! This release includes:
+
+- (_optional_) `<highlight of breaking changes>`
+- `<Summary of new features>` (_one or multiple bullets_)
+
+
+## Breaking changes
+
+- `#pr`, `<small_break_1>`
+- [websockets#60](https://github.com/grafana/xk6-websockets/pull/60) allows manual set `name` tag, which also overwrites `url` tag with `name` value. This is aligning of the logic that was implemented in the k6 v0.41. Thanks, @mkadirtan for contributing!
+
+### (_optional h3_) `<big_breaking_change>` `#pr`
+
+## New features
+
+### Add support for uploading files from the browser module [browser#1097](https://github.com/grafana/xk6-browser/pull/1097), [browser#1244](https://github.com/grafana/xk6-browser/pull/1244)
+
+You can now upload files using the available input forms on the website under test. The new API is `setInputFiles` which can be called from a `page`, `frame` or `elementHandle` types. It can upload one or more files encoded in the test script. To upload files from the local file system, work with the [experimental fs module](https://grafana.com/docs/k6/latest/javascript-api/k6-experimental/fs/).
+
+<details>
+<summary>Expand to see the examples.</summary>
+
+For the following examples, we will use the HTML file:
+
+```html
+<html>
+
+<body>
+    <form method="POST" action="/upload" enctype="multipart/form-data">
+        <input type="file" name="upl" id="upload" multiple />
+        <input type="submit" value="Send" />
+    </form>
+</body>
+
+</html>
+```
+
+Uploading a file can be achieved with the following script:
+
+```js
+// Import the k6 encoder module.
+import encoding from 'k6/encoding';
+...
+export default async function () {
+  const page = browser.newPage();
+
+  await page.goto(url)
+
+  // Encode and upload some data into a plain text file called test.txt.
+  page.setInputFiles('input[id="upload"]', { name: 'test.txt', mimetype: 'text/plain', buffer: encoding.b64encode('Hello World') })
+  
+  // Click on the submit button on the form to upload the file.
+  const submitButton = page.locator('input[type="submit"]')
+  await Promise.all([page.waitForNavigation(), submitButton.click()])
+
+  page.close();
+}
+```
+
+Uploading multiple files can be done with the use of an array:
+
+```js
+page.setInputFiles('input[id="upload"]',
+    [{ name: 'test.txt', mimetype: 'text/plain', buffer: encoding.b64encode('Hello World') },
+    { name: 'test.json', mimetype: 'text/json', buffer: encoding.b64encode('{"message": "Hello World"}') }])
+```
+
+</details>
+
+Thanks to @bandorko! :bow: :tada:
+
+### Introducing options.cloud [#3348](https://github.com/grafana/k6/pull/3348), [#3407](https://github.com/grafana/k6/pull/3407)
+
+In this release, we introduce a new way of defining cloud options. From now on, you can use just use `options.cloud` instead of the `options.ext.loadimpact`. 
```

Comment:
```suggestion
In this release, we introduce a new way of defining cloud options. From now on, you can use `options.cloud` instead of `options.ext.loadimpact`. 
```